src/main/java/lotto/io/OutputParser.java

```diff
@@ -0,0 +1,87 @@
+package lotto.io;
+
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+import lotto.dto.IncomeStatics;
+import lotto.dto.PrizeStatics;
+import lotto.dto.PurchasedLottos;
+
+public class OutputParser {
+
+    private static final String NEW_LINE = "\n";
+    private static final String PERCENT_FORMAT = "#,##0.0";
+
+    public String parsePurchasedLottos(final PurchasedLottos purchasedLottos) {
+        StringBuilder stringBuilder = new StringBuilder();
+
+        stringBuilder.append(purchasedLottos.lottos().size());
+        stringBuilder.append("개를 구매했습니다.");
+        stringBuilder.append(NEW_LINE);
+        stringBuilder.append(getPurchasedLottoString(purchasedLottos));
+
+        return stringBuilder.toString();
+    }
+
+    private static String getPurchasedLottoString(final PurchasedLottos purchasedLottos) {
+        StringBuilder stringBuilder = new StringBuilder();
+
+        purchasedLottos.lottos().forEach(lotto -> {
+            stringBuilder.append(lotto);
+            stringBuilder.append(NEW_LINE);
+        });
+
+        return stringBuilder.toString();
+    }
+
+    public String parsePrizeStatics(final PrizeStatics prizeStatics) {
+        StringBuilder stringBuilder = new StringBuilder();
+
+        stringBuilder.append("당첨 통계");
+        stringBuilder.append(NEW_LINE);
+        stringBuilder.append("---");
+        stringBuilder.append(getPrizeCountString(prizeStatics));
+
+        return stringBuilder.toString();
+    }
+
+    private static String getPrizeCountString(final PrizeStatics prizeStatics) {
+        StringBuilder stringBuilder = new StringBuilder();
+
+        prizeStatics.prizeCount().forEach((prize, count) -> {
+            stringBuilder.append(NEW_LINE);
+            stringBuilder.append(prize.description);
+            stringBuilder.append(" - ");
+            stringBuilder.append(count);
+            stringBuilder.append("개");
+        });
+
+        return stringBuilder.toString();
+    }
+
+    public String parseIncomeStatics(final IncomeStatics incomeStatics) {
+        StringBuilder stringBuilder = new StringBuilder();
+
+        stringBuilder.append("총 수익률은 ");
+        stringBuilder.append(formatPercent(incomeStatics.incomePercent()));
+        stringBuilder.append("%입니다.");
+
+        return stringBuilder.toString();
```

Comment:
```suggestion
        return String.format("총 수익률은 %,.1f%%입니다.", incomeStatics.incomePercent())
```

 혹시 요렇게 표현하는건 어떻게 생각하시나요?!

Reply:
저도 이 방식 고민했었는데, `DecimalFormat`이 좀 더 잘 읽힌다는 생각이 있었습니다. ㅎㅎ
제가 이 방식에 익숙하지 않은 탓일 수도 있어요!

Reply:
@Dompoo 
 앗 그렇군요! `DecimalFormat` 은 처음 보는거라 덕분에 알아갑니다!